pre-commit.sh

```diff
@@ -23,7 +23,7 @@ function abort() {
 trap abort ERR
 trap cleanup EXIT
 
-files=($(git status --porcelain draft-* rfc* | sed '/^[MARCU]/{s/.*\(draft-\|rfc\)/\1/;p;};d' | sort))
+files=($(git status --porcelain draft-* rfc* | sed '/^[[:space:]]*[MARCU]/{s/[[:space:]]*[MARCU][[:space:]]*\(draft-\|rfc\)/\1/;p;};d' | sort))
```

Comment:
How about a simpler approach:

```suggestion
files=($(git status --porcelain draft-* rfc* | sed '/^[MARCU]/{s/.* \(draft-\|rfc\)/\1/;p;};d' | sort))
```

The man-page for `git status` indicates that the fields are separated by a single space, so this should work without making this too complicated.

Reply:
Yes, that should work.  The non-porcelain output has more whitespace there, but this should work for porcelain.